spec/test-utils.js

```diff
@@ -212,18 +212,18 @@ MockStorageApi.prototype = {
  * @returns {Promise} promise which resolves (to `event`) when the event has been decrypted
  */
 export function awaitDecryption(event) {
-    if (!event.isBeingDecrypted()) {
-        return Promise.resolve(event);
-    }
-
-    logger.log(`${Date.now()} event ${event.getId()} is being decrypted; waiting`);
+    if (event.getClearContent() !== null) {
```

Comment:
Because this PR does not proactively trigger decryption for all events. 
It is not possible that an event is encrypted and not in the process of being decrypted.

`getClearContent` is the function that returns the decrypted content of an event. I'll add some code comment to clarify